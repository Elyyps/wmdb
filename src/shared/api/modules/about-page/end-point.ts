import { generateAboutPageDummyData } from "./generate-dummy-data";

export const generateAboutPageData = generateAboutPageDummyData;
