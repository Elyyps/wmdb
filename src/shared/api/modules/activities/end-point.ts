import { generateActivitiesDummyData } from "./dummy-data";

export const generateActivitiesData = generateActivitiesDummyData;
