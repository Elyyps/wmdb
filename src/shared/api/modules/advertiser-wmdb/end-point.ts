import { generateAdvertiserDummyData } from "./dummy-data";
export const generateAdvertiserData = generateAdvertiserDummyData;
