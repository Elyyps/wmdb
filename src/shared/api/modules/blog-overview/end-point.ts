import { generateBlogOverviewDummyData, generateCards } from "./dummy-data";
import { ICompanyOutingCard } from "@app/api/core/company-outing-card";

export const generateBlogOverviewData = generateBlogOverviewDummyData;
const amountOfCards = 50;
export const getCardsWithPagination = (skip: number, take: number): ICompanyOutingCard[] => {
  let cards: ICompanyOutingCard[] = generateCards(amountOfCards);
  cards = cards.slice(skip, take + skip);

  return cards;
};
