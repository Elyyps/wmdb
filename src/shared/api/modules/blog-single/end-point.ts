import { generateBlogSingleDummyData } from "./dummy-data";

export const generateBlogSingleData = generateBlogSingleDummyData;
