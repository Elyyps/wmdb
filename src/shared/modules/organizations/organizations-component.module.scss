.organizations {
  max-height: calc(100% - 534px);
  @media (max-width: $breakpoint-small) {
    max-height: calc(100% - 702px);
  }
  &:not(:first-child) {
    margin-top: 16px;
  }

  &__list {
    margin-top: 6px;
    font-size: 12px;
    display: flex;
    flex-wrap: wrap;

 
  }
  li,
  h5 {
    width: 33%;
    margin-bottom: 12px;
    text-decoration: underline;
    @media (max-width: $breakpoint-medium) {
    }
    @media (max-width: $breakpoint-small) {
    }
    @media (min-width: $breakpoint-small) {
      margin-bottom: 9px;
    }
  }
  a {
    width: auto;
    color: $gray;
  }
}
