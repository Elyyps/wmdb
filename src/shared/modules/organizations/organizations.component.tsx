import * as React from "react";
import styles from "./organizations-component.module.scss";
import { ILocation } from "@app/api/core/location";
import { LinkComponent } from "@app/core/link";
import { ILink } from "@app/api/core/link";
import { ModuleSectionComponent } from "@app/core/module-section";

export interface IOrganizationsComponentProps {
  locations: ILocation[];
}

const OrganizationsComponent = ({ locations }: IOrganizationsComponentProps) => (
  <ModuleSectionComponent paddingTop={"40px"} paddingTopMobile="32px" paddingBottom={"40px"}>
    {locations.map((location: ILocation, key: number) => (
      <div className={styles["organizations"]} key={key}>
        <h5>{location.title}</h5>
        {location.links && (
          <ul className={styles["organizations__list"]}>
            {location.links.map((link: ILink, i: number) => (
              <li key={i}>
                <LinkComponent to={link.url}>{link.title}</LinkComponent>
              </li>
            ))}
          </ul>
        )}
      </div>
    ))}
  </ModuleSectionComponent>
);

export { OrganizationsComponent };
