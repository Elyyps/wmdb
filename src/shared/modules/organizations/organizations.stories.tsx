import * as React from "react";

import { storiesOf } from "@storybook/react";
import { withA11y } from "@storybook/addon-a11y";
import { withProvider } from "@app/util";
import { OrganizationsComponent } from "./organizations.component";
import { generateOrganizationsData } from "@app/api/modules/organizations/end-point";

storiesOf("Organizations", module)
  .addDecorator(withA11y)
  .addDecorator(withProvider)
  .add("Basic implementation", () => <OrganizationsComponent locations={generateOrganizationsData()} />);
