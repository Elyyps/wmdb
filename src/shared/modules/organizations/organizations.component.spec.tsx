import React from "react";
import { shallow } from "enzyme";
import { OrganizationsComponent } from "./organizations.component";
import { generateOrganizationsData } from "@app/api/modules/organizations/end-point";

describe("[Organizations]", () => {
  it("should render component without crashing", () => {
    const renderedComponent = shallow(<OrganizationsComponent organisations={generateOrganizationsData()} />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
