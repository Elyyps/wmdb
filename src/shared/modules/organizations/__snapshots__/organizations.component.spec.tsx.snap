// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Organizations] should render component without crashing 1`] = `
<ModuleSectionComponent
  paddingBottom="40px"
  paddingTop="40px"
  paddingTopMobile="32px"
>
  <div
    className="organizations"
    key="0"
  >
    <h5>
      <LinkComponent
        to="/overzicht"
      >
        Art of Events Productions
         
      </LinkComponent>
    </h5>
    <ul
      className="organizations__list"
    >
      <li
        key="0"
      >
        <LinkComponent
          to="/adverteren"
        >
          U Deugt Wij Deugen
        </LinkComponent>
      </li>
      <li
        key="1"
      >
        <LinkComponent
          to="/adverteren"
        >
          The Ultimate Expedition
        </LinkComponent>
      </li>
      <li
        key="2"
      >
        <LinkComponent
          to="/adverteren"
        >
          Het Mislukte Feest
        </LinkComponent>
      </li>
      <li
        key="3"
      >
        <LinkComponent
          to="/adverteren"
        >
          Wie zijn de Mollen?
        </LinkComponent>
      </li>
      <li
        key="4"
      >
        <LinkComponent
          to="/adverteren"
        >
          Bosgolf
        </LinkComponent>
      </li>
    </ul>
  </div>
  <div
    className="organizations"
    key="1"
  >
    <h5>
      <LinkComponent
        to="/overzicht"
      >
        Astron37
         
      </LinkComponent>
    </h5>
    <ul
      className="organizations__list"
    >
      <li
        key="0"
      >
        <LinkComponent
          to="/adverteren"
        >
          Urban Walkabout; ... haal jij de 5?
        </LinkComponent>
      </li>
      <li
        key="1"
      >
        <LinkComponent
          to="/adverteren"
        >
          Een tactisch teamspel met een hoge fun-factor
        </LinkComponent>
      </li>
      <li
        key="2"
      >
        <LinkComponent
          to="/adverteren"
        >
          Toss & Throw (in- & outdoor)
        </LinkComponent>
      </li>
      <li
        key="3"
      >
        <LinkComponent
          to="/adverteren"
        >
          Survive the Never Never (in- & outdoor)
        </LinkComponent>
      </li>
      <li
        key="4"
      >
        <LinkComponent
          to="/adverteren"
        >
          Uitdagende game met een hoge fun-factor
        </LinkComponent>
      </li>
      <li
        key="5"
      >
        <LinkComponent
          to="/adverteren"
        >
          Spannende games voor iedereen (in- & outdoor)
        </LinkComponent>
      </li>
      <li
        key="6"
      >
        <LinkComponent
          to="/adverteren"
        >
          Pioneers Quest (in- & outdoor)
        </LinkComponent>
      </li>
      <li
        key="7"
      >
        <LinkComponent
          to="/adverteren"
        >
          Down Under Culture (in- & outdoor)
        </LinkComponent>
      </li>
      <li
        key="8"
      >
        <LinkComponent
          to="/adverteren"
        >
          Down Under Winter Games
        </LinkComponent>
      </li>
    </ul>
  </div>
  <div
    className="organizations"
    key="2"
  >
    <h5>
      <LinkComponent
        to="/overzicht"
      >
        Toon vertier
         
      </LinkComponent>
    </h5>
    <ul
      className="organizations__list"
    >
      <li
        key="0"
      >
        <LinkComponent
          to="/adverteren"
        >
          Dagje Terug in de Tijd
        </LinkComponent>
      </li>
      <li
        key="1"
      >
        <LinkComponent
          to="/adverteren"
        >
          Dagje uit- Verassingsdag
        </LinkComponent>
      </li>
      <li
        key="2"
      >
        <LinkComponent
          to="/adverteren"
        >
          Dagje Tukkend Touren
        </LinkComponent>
      </li>
      <li
        key="3"
      >
        <LinkComponent
          to="/adverteren"
        >
          Casino Speelavond All-in
        </LinkComponent>
      </li>
      <li
        key="4"
      >
        <LinkComponent
          to="/adverteren"
        >
          Sensationele fotopuzzeltocht
        </LinkComponent>
      </li>
      <li
        key="5"
      >
        <LinkComponent
          to="/adverteren"
        >
          Dagje Brabant
        </LinkComponent>
      </li>
      <li
        key="6"
      >
        <LinkComponent
          to="/adverteren"
        >
          Wie is de Mol?
        </LinkComponent>
      </li>
      <li
        key="7"
      >
        <LinkComponent
          to="/adverteren"
        >
          Cold Case
        </LinkComponent>
      </li>
      <li
        key="8"
      >
        <LinkComponent
          to="/adverteren"
        >
          Tuk Tuk & Meermansfiets Special
        </LinkComponent>
      </li>
      <li
        key="9"
      >
        <LinkComponent
          to="/adverteren"
        >
          Nostalgische Solexdag
        </LinkComponent>
      </li>
      <li
        key="10"
      >
        <LinkComponent
          to="/adverteren"
        >
          Tuk escape
        </LinkComponent>
      </li>
      <li
        key="11"
      >
        <LinkComponent
          to="/adverteren"
        >
          Bosescape
        </LinkComponent>
      </li>
      <li
        key="12"
      >
        <LinkComponent
          to="/adverteren"
        >
          Diner game
        </LinkComponent>
      </li>
      <li
        key="13"
      >
        <LinkComponent
          to="/adverteren"
        >
          Bos escape plus
        </LinkComponent>
      </li>
      <li
        key="14"
      >
        <LinkComponent
          to="/adverteren"
        >
          Op de vlucht
        </LinkComponent>
      </li>
      <li
        key="15"
      >
        <LinkComponent
          to="/adverteren"
        >
          Crazy 44
        </LinkComponent>
      </li>
      <li
        key="16"
      >
        <LinkComponent
          to="/adverteren"
        >
          Hunted
        </LinkComponent>
      </li>
      <li
        key="17"
      >
        <LinkComponent
          to="/adverteren"
        >
          Hilarische diner game
        </LinkComponent>
      </li>
      <li
        key="18"
      >
        <LinkComponent
          to="/adverteren"
        >
          Spetterende spellentocht
        </LinkComponent>
      </li>
    </ul>
  </div>
</ModuleSectionComponent>
`;
